### PR TITLE
PSMDB-971 Implement KMIP encryption (4.4) 

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
modified: data-at-rest-encryption.rst
new file: kmip.rst

### CODE REVIEW

File: source/data-at-rest-encryption.rst [hunk 1]
```diff
@@ -15,8 +15,7 @@ files can be decrypted and read by parties with the decryption key.
 
  The |feature| in |PSMDB| is introduced in version 3.6 to be compatible with
  |feature| interface in |mongodb|. In the current release of |PSMDB|, the |feature| does
- not include support for |abbr.kmip|, or |amazon-aws| key management
- services. Instead, |PSMDB| is :ref:`integrated with HashiCorp Vault <vault>` for key management services. 
+ not include support for |amazon-aws| key management service. Instead, |PSMDB| is :ref:`integrated with HashiCorp Vault <vault>` and supports the secure transfer of keys using :ref:`Key Management Interoperability Protocol (KMIP) <kmip>`.
```
Comment (nastena1606) [inline source/data-at-rest-encryption.rst]:
```suggestion
 not include support for |amazon-aws| key management service. Instead, |PSMDB| is :ref:`integrated with HashiCorp Vault <vault>`. Starting with version 4.4.13-13, |PSMDB| supports the secure transfer of keys using :ref:`Key Management Interoperability Protocol (KMIP) <kmip>`.  This allows users to store encryption keys in their favorite KMIP-compatible key manager. when they set up encryption at rest.
```

**Review by igorsol — APPROVED** [review]
LGTM

File: source/kmip.rst [hunk 2]
```diff
@@ -0,0 +1,60 @@
+.. _kmip:
+
+Using the Key Management Interoperability Protocol (KMIP) 
+============================================================
+
+This feature is **technical preview** quality.
```
Comment (spron-in) [inline source/kmip.rst]:
```suggestion
This feature is in **technical preview** stage.
```

Comment (nastena1606) [reply source/kmip.rst]: Fixed

**Review by olexandr-havryliak — APPROVED** [review]

**Pull request merged**